analysis/getEfficienciesLbL.cpp

```diff
@@ -73,19 +87,24 @@ int main()
     auto photonsPassing = event->GetPhysObjects(kGoodPhoton);
     
     if(goodGenPhotons.size() != 2) continue;
-    nGenEvents++;
-    hists["reco_id_eff_den"]->Fill(goodGenPhotons.front()->GetEt());
-    
-    
-    TLorentzVector diphoton = physObjectProcessor.GetDiphoton(*goodGenPhotons[0], *goodGenPhotons[1]);
+    nGenEvents++;  
+    hists["reco_eff_den"]->Fill(goodGenPhotons.front()->GetEt());
+    hists_eta["reco_eff_eta_den"]->Fill(goodGenPhotons.front()->GetEta());
+ 
+    TLorentzVector diphoton;    
+    //TLorentzVector diphoton = physObjectProcessor.GetDiphoton(*goodGenPhotons[0], *goodGenPhotons[1]);
     
     // Check properties of this event
-    bool hasLbLTrigger            = event->HasTrigger(kDoubleEG2noHF);
+    bool hasLbLTrigger            = event->HasTrigger(kDoubleEG2noHF) || event->HasTrigger(kSingleEG3noHF) || event->HasTrigger(kSingleEG5noHF); 
```

Comment:
Are we sure we want to accept events with single EG triggers? For analysis we only use double EG, right?

Reply:
If we get more stats including singeEG3 and singleEG5 triggers, we should use them for analysis. In 2015, there was no changes in number of events by includig singleEG5 trigger therefore we used only DoubleEG2 for analysis.

Reply:
Ok, then this is something we should verify as well -> adding to Trello.